buildkitd/entrypoint.sh

```diff
@@ -38,19 +31,51 @@ if [ "$EARTHLY_RESET_TMP_DIR" = "true" ]; then
     rm -rf "${EARTHLY_TMP_DIR:?}"/* || true
 fi
 
-if ! lsmod | grep -wq "^ip_tables"; then
-  echo "Legacy ip_tables is not loaded."
+if [ -z "$IP_TABLES" ]; then
+    echo "Autodetecting iptables"
+
+    if lsmod | grep -wq "^ip_tables"; then
+        echo "Detected iptables-legacy module"
+        IP_TABLES="iptables-legacy"
+
+    elif lsmod | grep -wq "^nf_tables"; then
+        echo "Detected iptables-nft module"
+        IP_TABLES="iptables-nft"
+    else
+        echo "Could not find an ip_tables module. Using heuristics."
```

Comment:
How about re-wording this to a single sentence:

```suggestion
        echo "Could not find an ip_tables module; falling back to heuristics."
```

that way it's less likely a user will misread it as `Could not find an ip_tables module using heuristics.`.

Reply:
Sure. I question people missing punctuation though :P